src/table/cells/handle-cell-focus.js

```diff
@@ -11,33 +11,74 @@ export const updateFocus = (rowElements, cellCoord, shouldFocus = true) => {
   }
 };
 
-export const handleClickToFocusBody = (cell, focusedCellCoord, rootElement) => {
+export const findCellWithTabStop = (rootElement) => rootElement.querySelector("td[tabindex='0'], th[tabindex='0']");
+
+export const handleClickToFocusBody = (cell, rootElement, setfocusedCellCoord, constraints) => {
+  if (constraints.active) return;
```

Comment:
No need to do this. If we consider being able to focus a cell in edit mode a bug, we can remove it, but in a different PR. But it might actually be useful (the old table does it)

Reply:
✅ ok, that is reasonable, i'll remove it